app/views/hyrax/base/_citation_and_copy_button.html.erb

```diff
@@ -0,0 +1,16 @@
+<div id=<%= "#{citation_style}-citation-header" %>>
+    <div class="row">
+        <div class="col-sm-2">
+            <h3><%= t("#{citation_style}_header") %></h3>
+        </div>
+        <div class="col-sm-1 copy-button">
+            <button id=<%= "#{citation_style}-copy-button" %> class="btn btn-primary" data-clipboard-target= <%="##{citation_style}-citation-text" %> data-clipboard-action="copy" data-toggle="tooltip" data-placement="bottom">
```

Comment:
Then just add the `.citation-copy-button` class here and on all the other buttons